src/main/java/com/google/sps/servlets/InfoCardServlet.java

```diff
@@ -55,115 +51,45 @@
    */
   @Override
   public void doPut(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
Just curious, are we expecting to have the cron job call this endpoint? If so, would it be possible to combine the two calls together?  I think that way we won't need to store Civic api provided ID's in our datastore anymore too, to reduce confusion.

Reply:
The PUT requests are intended to be called separately, so doPut in ElectionServlet is called on the electionlist.html page, and stores all of the basic info about the list of elections. Then once an election is chosen doPut in InfoCardServlet is called on the electionInfo.html page and stores info about a specific election.